TOMP-API.yaml

```diff
@@ -1749,7 +1815,7 @@ components:
       properties:
         type:       
           type: string
-          enum: [VEHICLE_NOT_AVAILABLE, USER_NO_SHOW, OTHER]
+          enum: [VEHICLE_NOT_AVAILABLE, USER_NO_SHOW, ETA, OTHER]
```

Comment:
I think the ETA is weird here. For one thing, the description states that this notification is for issues. The other thing is that there is already a leg-progress object (line 1733 / 1799) and I see ETA more of an update of that.

Reply:
The ETA is before the taxi arrives at the pick up location.. The leg hasn't started yet.